### ISSUE TITLE
Array of Maps causes parsing errors in nested description Items->Types

### ISSUE THREAD

**Issue by NamelessHH** [swaggo/swag#584]
**Describe the bug**
`
\u0026{%!s(token.Pos=222) string %!s(*ast.InterfaceType=\u0026{233 0xc00039a300 false})}
`

shows as the description to an array type of map.

**To Reproduce**

1. create a datatype 
```
type DataDto struct { 
	Data           []map[string]interface{} `json:"data"` //The Individual Records
	Count          int                      `json:"count"`
	Average   float64                  `json:"averageScore,string"`
}
```
2. set it ass sucess type `// @Success 200 {object} handler.DataDto`
3. create swag documentation `swag init`
4. the below shows as the description 
```
                "data": {
                    "description": "The Individual Records",
                    "type": "array",
                    "items": {
                        "type": "\u0026{%!s(token.Pos=222) string %!s(*ast.InterfaceType=\u0026{233 0xc00039a300 false})}"
                    }
                },
```

**Expected behavior**
should be in this case `"items":{"type":"object"}` 

**Screenshots**

**Your swag version**
e.g. 1.6.3

**Your go version**
e.g. 1.13.4

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - IDE: VSCode

**Additional context**
have tried []map[string]string as well and the type is different  but still wrong.

**Comment by sdghchj** [comment]
I've just commit a PR for map.
Let me PR again for your issue.

**Comment by easonlin404** [comment]
@NamelessHH fixed by #586. please reopen it or raise a new issue if there are any problems.
